solution/generate_slot_solution.go

```diff
@@ -80,7 +80,7 @@ func GenerateSlotSolution(timeMatcher *matching.TimeMatcher, location string, ev
 	}
 
 	cityCountry := strings.Split(location, ",")
-	evTags := make([]string, len(stayTimes))
```

Comment:
previously this size allocation is confusing, even though at this point `len(stayTimes)==len(evTags`